src/lib.rs

```diff
@@ -51,15 +51,16 @@ pub use crate::error::Error;
 
 use serde_json::value::RawValue;
 
-/// Shorthand method to convert an argument into a [Box<serde_json::value::RawValue>].
-/// Since serializers rarely fail, it's probably easier to use [arg] instead.
+/// Shorthand method to convert an argument into a [`Box<serde_json::value::RawValue>`].
+///
+/// Since serializers rarely fail, it's probably easier to use [`arg`] instead.
```

Comment:
Just `arg`, no? It's not a identifier so it can't be a link.

Reply:
This one is correct, `arg` is a function as well. This threw me too, perhaps we should change the paramater identifier to not be `arg`?

Reply:
Ooooooooohhhh. :D